### PR TITLE
Fogbow test review

### PR DESCRIPTION
**Pull request by ANKerD**

### CODE REVIEW

File: src/main/java/cloud/fogbow/fns/utils/AgentCommunicatorUtil.java [hunk 1]
```diff
@@ -29,7 +29,7 @@ public static boolean createFederatedNetwork(String cidrNotation, String virtual
             LOGGER.info(String.format(Messages.Error.TRYING_TO_CREATE_AGENT_OUTPUT, cidrNotation,
                     ProcessUtil.getOutput(process)));
             LOGGER.info(String.format(Messages.Error.TRYING_TO_CREATE_AGENT_ERROR, cidrNotation, ProcessUtil.getError(process)));
-            resultCode = process.waitFor();
+            resultCode = process.waitfor ();
```
Comment (thiagomanel) [inline src/main/java/cloud/fogbow/fns/utils/AgentCommunicatorUtil.java]: remove this empty space

File: src/main/java/cloud/fogbow/fns/utils/AgentCommunicatorUtil.java [hunk 2]
```diff
@@ -57,7 +57,7 @@ public static boolean deleteFederatedNetwork(String cidr) {
             LOGGER.info(String.format(Messages.Error.TRYING_TO_DELETE_AGENT_OUTPUT, cidr,
                     ProcessUtil.getOutput(process)));
             LOGGER.info(String.format(Messages.Error.TRYING_TO_DELETE_AGENT_ERROR, cidr, ProcessUtil.getError(process)));
-            resultCode = process.waitFor();
+            resultCode = process.waitfor ();
```
Comment (thiagomanel) [inline src/main/java/cloud/fogbow/fns/utils/AgentCommunicatorUtil.java]: remove this empty space

File: src/main/java/cloud/fogbow/fns/utils/AgentCommunicatorUtil.java (hunk 1 above)
Comment (ANKerD) [reply src/main/java/cloud/fogbow/fns/utils/AgentCommunicatorUtil.java]: Fixed.

File: src/main/java/cloud/fogbow/fns/utils/AgentCommunicatorUtil.java (hunk 2 above)
Comment (ANKerD) [reply src/main/java/cloud/fogbow/fns/utils/AgentCommunicatorUtil.java]: Also fixed.

**Pull request merged**